### DISCUSSION TITLE
Is the website still up to date?

### DISCUSSION THREAD

**Discussion by blue-eric** [teslamate-org/teslamate#3946] [General]
Hello, I want to know about this website https://docs.teslamate.org/is the code still updated?

**Comment by cwanja** [comment]
That specific link is a 404, but besides that yes the docs website is up to date. 

What is not working or looks wrong?

**Reply by blue-eric** [reply]
Many conditional formats of homeassistant have been adjusted, including MQTT, but it seems that the code on the webpage is still using the old standard, so I wonder if this webpage is not updated, or is it not modified synchronously?
